src/plugins/ini/ini.c

```diff
@@ -1544,10 +1511,10 @@ int elektraIniSet (Plugin * handle, KeySet * returned, Key * parentKey)
 Plugin * ELEKTRA_PLUGIN_EXPORT (ini)
 {
 	// clang-format off
-	return elektraPluginExport ("ini",
```

Comment:
can we turn on clang-format for this part too ? 
while coding i often use vims reformating (`gg=G`) which causes these problems